dotnet/samples/04.ai.d.chainedActions.listBot/ListBot.csproj

```diff
@@ -0,0 +1,49 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+    <Nullable>enable</Nullable>
+    <ImplicitUsings>enable</ImplicitUsings>
+  </PropertyGroup>
+
+  <ItemGroup>
+	<ProjectCapability Include="TeamsFx" />
+  </ItemGroup>
+	
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Bot.Builder" Version="4.20.0" />
```

Comment:
Could *Microsoft.Bot.Builder* and *Microsoft.Bot.Connector* be removed since already included by TeamsAI SDK?

Reply:
Removed.